### ISSUE TITLE
Allow or document how to use publishToMavenLocal without signing

### ISSUE THREAD

**Issue by jmini** [asciidoctor/asciidoctorj#608]
I have tried to run:

    ./gradlew publishToMavenLocal

And I got this exception:

```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':asciidoctorj:signPom'.
> Cannot perform signing task ':asciidoctorj:signPom' because it has no configured signatory

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

Maybe this is already possible with the current build script, but I did not find how to do it.

I am really a beginner with gradle.

**Comment by abelsromero** [comment]
Just need to disable signing adding a project property:
https://github.com/asciidoctor/asciidoctorj/blob/c4b31fdeceeb62ebcad3d8b787b92361c26050a1/gradle/signing.gradle#L1

You can set in the shell (wiht -P) or add it to your global config ~/.gradle/gradle.properties (which is what I do).

Probably, this should be added to the `Develop in an IDE` section.

**Comment by jmini** [comment]
Thank you a lot. This works for me:

    ./gradlew publishToMavenLocal -Pskip.signing

I will propose a pull request for the readme file.

**Comment by jmini** [comment]
I consider this closed with pull request #627
